src/public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="description" content="You app for smart urban transport">
+    <meta name="keywords" content="Transport, ride, urban, car, share">
+    <meta name="author" content="Hege Refsnes">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
```

Comment:
good catch, added this to test a bootstrap component and forgot to import the css dependency using npm.